builtin/v13/miner/miner_types.go

```diff
@@ -394,3 +394,105 @@ type GetPeerIDReturn struct {
 type GetMultiAddrsReturn struct {
 	MultiAddrs []byte
 }
+
+// ProveCommitSectors3Params represents the parameters for proving committed sectors.
+type ProveCommitSectors3Params struct {
+	SectorActivations          []SectorActivationManifest
+	SectorProofs               [][]byte
+	AggregateProof             []byte
+	AggregateProofType         abi.RegisteredAggregationProof
```

Comment:
```suggestion
	AggregateProofType         *abi.RegisteredAggregationProof
```

probably, with FIP update, https://github.com/filecoin-project/builtin-actors/pull/1511

Reply:
Will be a separate update when that PR lands